tea-config-generator/README.md

```diff
@@ -0,0 +1,68 @@
+# CMR TEA Configuration Generator
+A lambda project to generate [Thin Egress App][teacode] (TEA) configuration files using [Serverless][sls]
+
+## Overrview
+A set of lambda functions to run under the CMR domain name which generate a TEA configuration.
+
+See: [Cumulus thin egress app][tea]
+
+## Usage
+
+The [run.sh](run.sh) command will execute many of the stages of the software needed
+get the application started. To see the options supported, run `./run.sh -h`. To
+start the application, run in one terminal window: `./run.sh -o`. This will install
+the lambda functions and start the offline server. In another terminal window try
+the following commands:
+
+    curl 'http://localhost:3000/dev/configuration/tea/capabilities'
+    curl -I 'http://localhost:3000/dev/configuration/tea/status'
+    curl -H 'Authorization: Bearer uat-token-value' 'http://localhost:3000/dev/configuration/tea/provider/POCLOUD'
+
+| Action       | URL                                       | Description |
+| ------------ | ----------------------------------------- | ----------- |
+| Capabilities | /configuration/tea/                       | Describe the service |
+| Capabilities | /configuration/tea/capabilities           | Describe the service |
+| Status       | /configuration/tea/status                 | Returns the service status |
+| Generate     | /configuration/tea/provider/{provider-id} | Generate the TEA configuration file |
+
+## Building
+
+1. Install serverless using one of these commands:
+    * `npm install -g serverless`
+    * `curl -o- -L https://slss.io/install | bash`
+2. Plugin: `serverless plugin install -n serverless-python-requirements`
+3. Plugin: `serverless plugin install -n serverless-s3-local`
+4. Python dependencies: `pip3 install -r requirements.txt`
+5. Install locally: `serverless offline`
+6. Run some example URLS: `./run.sh -e`
+
+### Docker
+
+The Dockerfile defines a node image which includes python3 and serverless for
+use as a build and deployment environment when running under a CI/CD system.
+
+## Deploying to AWS
+
+1. Create a role with a title like `tea-config-<env>-<location>-lambda-role`
```

Comment:
Update this section now that you are granting the IAM role in serverless.yml.

Reply:
your right, will do that